src/resource/resource-opening-hours/opening-period/OpeningPeriod.tsx

```diff
@@ -83,9 +84,22 @@ export default function OpeningPeriod({
           </button>
         </div>
         <ConfirmationModal
-          onConfirm={(): void => {
+          onConfirm={async (): Promise<void> => {
             if (datePeriod.id) {
-              deletePeriod(datePeriod.id);
+              try {
+                await deletePeriod(datePeriod.id);
+                toast.success({
+                  label: 'Aukiolo poistettu onnistuneesti',
+                  text: `Aukiolo ${datePeriod.id} poistettu onnistuneesti`,
```

Comment:
Im not sure if we should show database IDs in toast messages? Are those IDs shown somewhere else? I wonder if people understand what they are 🤔 . Maybe we could show the period title there instead?

Reply:
True, I can change this one